src/features/governance/updateProposalsInDB.ts

```diff
@@ -0,0 +1,271 @@
+/* eslint no-console: 0 */
+
+import { governanceABI } from '@celo/abis';
+import { and, eq, inArray, sql } from 'drizzle-orm';
+import database from 'src/config/database';
+import { eventsTable, proposalsTable } from 'src/db/schema';
+import { Chain, parseEventLogs, PublicClient, Transport } from 'viem';
+
+import { Addresses } from 'src/config/contracts';
+import { fetchProposalsFromRepo } from 'src/features/governance/fetchFromRepository';
+import { ProposalMetadata, ProposalStage } from 'src/features/governance/types';
+
+import '../../vendor/polyfill.js';
+
+export default async function updateProposalsInDB(
```

Comment:
this is a wild one. 

can we separate out. data collection. data manipulation. and data insertion?